### ISSUE TITLE
Possible incompat with Hyper Heisting

### ISSUE THREAD

**Issue by ImperatorStorm** [offyerrocker/PD2-AdvancedCrosshairs#1]
Using ACH with Hyper Heisting tends to crash at random points
my crashlog:
```
Application has crashed: C++ exception
mods/PD2-AdvancedCrosshairs/lua/menumanager.lua:3368: attempt to index local 'equipped_unit' (a nil value)



SCRIPT STACK

func() @mods/PD2-AdvancedCrosshairs/lua/playermovementstate.lua:2
enter() @mods/base/req/core/Hooks.lua:268
enter() lib/units/beings/player/states/playerstandard.lua:199
change_state() lib/units/beings/player/playermovement.lua:249
_change_player_state() lib/managers/playermanager.lua:985
set_player_state() lib/managers/playermanager.lua:943
clbk() lib/units/beings/player/states/playertased.lua:489
_update_queued_tasks() @mods/PD2-Hyper-Heisting/lua/enemymanager.lua:148
update() lib/managers/enemymanager.lua:51
update() lib/setups/gamesetup.lua:723
_update() lib/setups/networkgamesetup.lua:21
update() @mods\PocoHud3\poco\common.lua:310
core/lib/setups/coresetup.lua:557
```

another person's crashlog:
```
Application has crashed: C++ exception
mods/PD2-AdvancedCrosshairs/lua/menumanager.lua:3280: attempt to index local 'equipped_unit' (a nil value)



SCRIPT STACK

func() @mods/PD2-AdvancedCrosshairs/lua/playermovementstate.lua:2
enter() @mods/base/req/core/Hooks.lua:268
enter() lib/units/beings/player/states/playerstandard.lua:199
change_state() lib/units/beings/player/playermovement.lua:249
_change_player_state() lib/managers/playermanager.lua:985
set_player_state() lib/managers/playermanager.lua:943
clbk() lib/units/beings/player/states/playertased.lua:489
_update_queued_tasks() @mods/PD2-Hyper-Heisting/lua/enemymanager.lua:148
original() lib/managers/enemymanager.lua:51
update() @mods/base/req/core/Hooks.lua:265
update() lib/setups/gamesetup.lua:723
update() lib/setups/networkgamesetup.lua:21
core/lib/setups/coresetup.lua:557
```

**Comment by offyerrocker** [comment]
May I ask which version of ACH were you and the other person using? These appear seem to have the same root cause, but they point to different lines, and neither of them matches up with any version of ACH I'm using

**Comment by ImperatorStorm** [comment]
i'm guessing i was using `16 Nov 2020 v2`

no clue for the other guy

**Comment by offyerrocker** [comment]
Many thanks for the report/information. This should now be fixed as of commit 020b40b998caf228e6c1ea2f4d2003bccd7165da. You can either update to the latest development version on GitHub, or wait for the next stable release v4 (which should be out very soon :tm:). Please do let me know if this issue happens again, or if you have further issues!

**Comment by ImperatorStorm** [comment]
as of my limited testing, it is fixed.

I'll let you know if anything else pops up.

thanks!
